packages/contracts/test-suites/test-aave/eth-call-contracts.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { makeSuite, TestEnv } from "./helpers/make-suite";
+import { GetAllTrancheAssetsData } from "../../types";
+import { deployContract } from "../../helpers/contracts-helpers";
+
+makeSuite("Eth call contracts functionality", (testEnv: TestEnv) => {
+  it("Get all tranche assets data", async () => {
+    const { addressesProvider } = testEnv;
+
+    const c: GetAllTrancheAssetsData =
```

Comment:
no check on expected value?

Reply:
tbh this test does not work, ill remove it. it was purely to see if i can get more info on my error codes, and i did not. i will remove